src/kvstore/raftex/RaftPart.cpp

```diff
@@ -1198,6 +1204,7 @@ cpp2::ErrorCode RaftPart::verifyLeader(
     VLOG(2) << idStr_ << "The current role is " << roleStr(role_);
     UNUSED(lck);
     switch (role_) {
+        case Role::LEARNER:
```

Comment:
maybe learner can not verify leader, because when request vote, candidate do not send vote request to learner, so the learner do not know who is leader.

Reply:
That's ok.  If learner receive the logs from leader,  after check the term,   learn could follow the leader.

Reply:
yes, you are right.